src/test/kotlin/org/malachite/estella/people/seekers/DummyJobSeekerRepository.kt

```diff
@@ -21,6 +21,11 @@ class DummyJobSeekerRepository: JobSeekerRepository {
         return jobSeekers
     }
 
+    override fun findByUserMail(mail: String): Optional<JobSeeker> =
+        jobSeekers.firstOrNull { it.user.mail == mail }
+            ?.let { Optional.of(it) }
```

Comment:
Zawsze możesz zrobić zamiast tego `Optional.ofNullable(it)` i przed letem nie dawać pytajnika - co prawda nie do końca wiem, czym to się różni, ale może akurat coś poprawi